### DISCUSSION TITLE
Badge not appear.

### DISCUSSION THREAD

**Discussion by devflash101** [community/community#112466] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I've answered several questions in community but I can't get the Galaxy Brain badge till now.
Is it a bug? If yes, how to resolve this?

**Comment by queenofcorgis** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

**Reply by devflash101** [reply, marked as answer]
I understand the reason why I could not earn the badge with my past activity.
could you kindly tell me how to get that badge actually?
